README.pt-br.md

```diff
@@ -1,6 +1,6 @@
 ![](get.png)
 
-*Idiomas: [Inglês](README.md), Português Brasileiro (este aquivo)*
+*Idiomas: [English](README.md), [Brazilian Portuguese](README.pt-br.md),[Polish](README.pl.md).*
```

Comment:
Same here, If i am already reading portuguese readme, why do i need a link to this same portuguese readme?